lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1688,6 +1688,41 @@ void AtenSortIntOp::getCanonicalizationPatterns(RewritePatternSet &patterns,
   });
 }
 
+
+//===----------------------------------------------------------------------===//
+// AtenSortOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult AtenSortOp::fold(FoldAdaptor adaptor,
+                               SmallVectorImpl<OpFoldResult> &results) {
+  auto operand = getSelf();
+  auto operandTy = dyn_cast_if_present<ValueTensorType>(operand.getType());
```

Comment:
When would operand.getType() be nullptr? I'm wondering why you're using dyn_cast_if_present and not dyn_cast.

Reply:
good point, using dyn_cast for this and the other one